server.c

```diff
@@ -44,6 +48,25 @@ void handle_request_task(int fd, struct msg_request_task *req)
     write_msg(fd, &wrapper);
 }
 
+void log_task(struct msg_solution *solution) {
+    //we open the log file (create it if it doesn't already exist)
+    FILE *log_file = fopen(LOG_FILE, "a+");
+    if (log_file == NULL) {
+        fprintf(stderr, "Error opening task log file\n");
+        exit(1);
+    }
```

Comment:
Like the error checking, but worried about the whole server going down if this is not accessible...

Reply:
I have added the changes you mentioned. Let me know if I did them properly. Also i have removed the exit(1) line but how should we handle errors when it comes to opening the file or adding something to the file later?

Reply:
We might need to change its return type to an int later and check it during startup (I'm assuming this will happen after the command line parsing gets added -- so we'll keep it this way for now)